<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DruidTimer</class>
 <widget class="QGroupBox" name="DruidTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Time</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>40</number>
   </property>
   <item row="2" column="0">
    <widget class="QPushButton" name="_time_stop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hour</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="_time_hour"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Minutes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="_time_minutes"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="_time_start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="QLCDNumber" name="_time_lcd"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
